cognify/optimizer/core/flow.py

```diff
@@ -178,6 +182,10 @@ class OptConfig:
     use_SH_allocation: bool = field(default=False)
     patience: Optional[PatienceConfig] = field(default_factory=lambda: PatienceConfig(0.01,0.01,0.01,5))
     frac: float = field(default=1.0)
+    use_HB_allocation: bool = field(default=False)
+    use_SH_allocation: bool = field(default=False)
```

Comment:
what's the difference between HB and SH? I thought SH is just a part of HB. Also, why default is false for these two? shouldn't default be always enabling the one that's in the paper?

Reply:
Right. The major difference comes from whether the bucket size and initial budget are changed in each iteration. Changed to `static_SH`.